pom.xml

```diff
@@ -1315,16 +1362,79 @@
             <name>Dummy to avoid accidental deploys</name>
             <url />
         </repository>
+        <site>
+            <id>default</id>
+            <name>Mina SSHD</name>
+            <url>file://${project.build.directory}/deploy</url>
+        </site>
     </distributionManagement>
 
     <reporting>
-       <plugins>
-           <plugin>    <!-- used by PMD to create a source XREF -->
-               <groupId>org.apache.maven.plugins</groupId>
-               <artifactId>maven-jxr-plugin</artifactId>
-               <version>3.0.0</version>
-           </plugin>
-       </plugins>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-jxr-plugin</artifactId>
+                <version>3.0.0</version>
+            </plugin>
+            <plugin>
+                <groupId>org.jacoco</groupId>
+                <artifactId>jacoco-maven-plugin</artifactId>
+                <reportSets>
+                    <reportSet>
```

Comment:
As previously mentioned, I don't want `jacoco` to be part of the regular build cycle, but rather activated via a profile (`-Pjacoco`)